common/frame.go

```diff
@@ -1583,12 +1583,9 @@ func (f *Frame) WaitForLoadState(state string, opts goja.Value) {
 		k6Throw(f.ctx, "cannot parse waitForLoadState options: %v", err)
 	}
 
-	waitUntil := LifecycleEventLoad
-	switch state {
-	case "domcontentloaded":
-		waitUntil = LifecycleEventDOMContentLoad
-	case "networkidle":
-		waitUntil = LifecycleEventNetworkIdle
+	var waitUntil LifecycleEvent
+	if err = waitUntil.UnmarshalText([]byte(state)); err != nil {
+		k6Throw(f.ctx, "waitForLoadState error: %v", err)
```

Comment:
[Puppeteer actually defaults to `load` if `state` is not provided](https://playwright.dev/docs/api/class-page#page-wait-for-load-state), so that behavior shouldn't change. I'll fix it.